Add unit tests for client.py

mykhaly opened the pull request.

selurvedu commented on openprocurement_client/client.py:
```diff
@@ -302,12 +302,10 @@ def patch_award(self, tender, award):
         return self._patch_resource_item(
             '{}/{}/{}/{}/documents/{}'.format(
-                self.prefix_path, tender.data.id, "cancellations", tender['data']['cancellations'][cancel_num]['id'], tender['data']['cancellations'][cancel_num]['documents'][doc_num]['id']
+                self.prefix_path, tender.data.id, "cancellations", cancellation_id, cancellation_doc_id
```
Винеси це в окремий коміт.

mykhaly replied:
done

mykhaly committed:
Update patch_cancellation_document

Work with cancel and document ids instead of cancel and document numbers
Rename cancellation_data -> cancellation

mykhaly committed:
Add unit tests for client.py

mykhaly committed:
Add methods for complaints

mykhaly committed:
Resolve merge conflict
